scripts/add-new-bcd.ts

```diff
@@ -235,6 +235,7 @@ const main = async (): Promise<void> => {
     {addNewFeatures: true},
     bcd.browsers,
     overrides,
+    "",
```

Comment:
This doesn't seem to work? I get `Error: ENOENT: no such file or directory, open ''` when running `npm run add-new-bcd` on main.

Reply:
Thanks for catching this! Will submit a quick patch in a new PR!

Reply:
PR here: https://github.com/openwebdocs/mdn-bcd-collector/pull/995